packages/app/src/files-manager/components/files-manager/files-manager.component.tsx

```diff
@@ -58,11 +58,16 @@ const FilesManager: FunctionComponent<FilesManagerProps> = ({
           ...element,
           size: free,
         }
-      }
-      if (element.type === DiskSpaceCategoryType.UsedSpace) {
+      } else if (element.type === DiskSpaceCategoryType.UsedSpace) {
+        return {
+          ...element,
+          size: reservedSpace,
+        }
+      } else if (element.type === DiskSpaceCategoryType.Music) {
```

Comment:
Small thing, but I think it would be good to add a test for that. Seems to me that this mapping might get broken by accident (e.g. somebody deletes `filesAmount` by accident and TS does not report that this is an error) and it would be good to check if the right and expected properties are passed.

Reply:
Changed to more elegant way in [2f838ad](https://github.com/mudita/mudita-center/pull/915/commits/2f838ad5888146b962ea0c53a4419615d2c45ec8)